ci/spec.py

```diff
@@ -14,8 +14,8 @@ def spec(package: str, pyproject_toml_path: Path) -> str:
         pyproject_toml = tomllib.load(pyproject_toml_file)
     tools = pyproject_toml["project"]["optional-dependencies"]["tools"]
     package_specs = [spec for spec in tools if spec.split("==")[0] == package]
-    return package_specs[0] if package_specs else ""
+    return sep.join(package_specs[0].split("==")) if package_specs else ""
```

Comment:
This method is kinda dirty, can't we comprehend the toml file directly? The `tomllib` is a std lib since python 3.11

Reply:
We use tomllib to read the toml file, but the package specs are strings of the form `"package==x.y.z"`. tomllib doesn't help with that.